etcdctl/ctlv3/command/printer_simple.go

```diff
@@ -200,7 +200,7 @@ func (s *simplePrinter) RoleGet(role string, r v3.AuthRoleGetResponse) {
 		} else {
 			fmt.Printf("\t[%s, <open ended>", sKey)
 		}
-		if v3.GetPrefixRangeEnd(sKey) == sRangeEnd {
+		if v3.GetPrefixRangeEnd(sKey) == sRangeEnd && len(sKey) > 0 {
```

Comment:
When sKey is empty, `GetPrefixRangeEnd ` will return []byte{0}, it will be same with rootPerm rangeEnd []byte{0}.
And when sKey is empty, it shouldn't show prefix.

Reply:
Out of curiosity: What's wrong with showing 'empty' prefix. Prefixes are IMHO easier to understand than ranges.

Reply:
it will show：
```

Role root
KV Read:
        [, <open ended> (prefix)
KV Write:
        [, <open ended> (prefix)
```
It's not the expected when role is `root`

Reply:
In fact, the empty prefix is no sense.